src/tir/usmp/analysis/extract_buffer_info.cc

```diff
@@ -491,6 +559,55 @@ BufferInfoAnalysis BufferInfoExtractor::operator()(const PrimFunc& main_func) {
       open_set.erase(le_event.buffer_info);
     }
   }
+
+  // All items in RO pool should have conflicts with each other in this RO pool
```

Comment:
nit : lets stick to ConstantPools / ConstantPoolInfo

Reply:
amended